resources/assets/js/forms.js

```diff
@@ -435,3 +424,55 @@ ready(() => {
     }
   });
 });
+
+/**
+ * Live reload arrive stead instead of submitting form
+ */
+ready(() => {
+  document.querySelectorAll('form.arrive_form').forEach((element) => {
+    element.addEventListener('submit', async (e) => {
+      e.preventDefault();
+      e.stopPropagation();
+
+      const form = e.target;
+      const row = form.parentElement.parentElement;
+      const name = row.querySelector('.column_name a');
+      const arrived = row.querySelector('.column_arrived');
+      const arrival = row.querySelector('.column_rendered_arrival_date');
+      const button = form.querySelector('[name="send"]');
+      const data = new FormData(e.target);
+      data.append('send', '!');
+
+      const response = await fetch(window.location.href, {
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'X-CSRF-TOKEN': data.get('_token'),
+        },
+        body: data,
+      });
+
+      if (!response.ok) {
+        console.warn('Broken response, submitting form anyways');
+        form.submit();
+        return;
+      }
+
+      const body = await response.json();
+      arrival.innerText = body.arrival_date;
+      const newButton = new DOMParser().parseFromString(body.button, 'text/html').body.firstChild;
```

Comment:
That's kind of dangerous. You create a DOM element from maybe spoofed text.
I know it's paranoid but...

`const btn = document.createElement('button')` would do the job also. It would be more JS code to set all the button attributes, but more save